executor/aggfuncs/func_lead_lag_test.go

```diff
@@ -111,4 +112,54 @@ func (s *testSuite) TestLeadLag(c *C) {
 	for _, test := range tests {
 		s.testWindowFunc(c, test)
 	}
+
+}
+
+func (s *testSuite) TestMemLeadLag(c *C) {
+	zero := expression.NewZero()
+	one := expression.NewOne()
+	two := &expression.Constant{
+		Value:   types.NewDatum(2),
+		RetType: types.NewFieldType(mysql.TypeTiny),
+	}
+	three := &expression.Constant{
+		Value:   types.NewDatum(3),
+		RetType: types.NewFieldType(mysql.TypeTiny),
+	}
+	million := &expression.Constant{
+		Value:   types.NewDatum(1000000),
+		RetType: types.NewFieldType(mysql.TypeLong),
+	}
+
+	numRows := 3
+	tests := []windowMemTest{
+		// lag(field0, N)
+		buildWindowMemTesterWithArgs(ast.WindowFuncLag, mysql.TypeLonglong,
+			[]expression.Expression{zero}, 0, numRows, aggfuncs.DefPartialResult4LeadLagSize, rowMemDeltaGens),
+		buildWindowMemTesterWithArgs(ast.WindowFuncLag, mysql.TypeLonglong,
+			[]expression.Expression{one}, 0, numRows, aggfuncs.DefPartialResult4LeadLagSize, rowMemDeltaGens),
+		buildWindowMemTesterWithArgs(ast.WindowFuncLag, mysql.TypeLonglong,
+			[]expression.Expression{two}, 0, numRows, aggfuncs.DefPartialResult4LeadLagSize, rowMemDeltaGens),
+		buildWindowMemTesterWithArgs(ast.WindowFuncLag, mysql.TypeLonglong,
+			[]expression.Expression{three}, 0, numRows, aggfuncs.DefPartialResult4LeadLagSize, rowMemDeltaGens),
+		buildWindowMemTesterWithArgs(ast.WindowFuncLag, mysql.TypeLonglong,
+			[]expression.Expression{million}, 0, numRows, aggfuncs.DefPartialResult4LeadLagSize, rowMemDeltaGens),
+
+		// lead(field0,N)
```

Comment:
Please add a space here

Reply:
ok, Do you mean add space like `// lead(field0, N)`

Reply:
`// lead(field0, N)`
space between filed0, and N